### ISSUE TITLE
.info gives uninformative message for bbob-constrained functions

### ISSUE THREAD

**Issue by brockho** [numbbo/coco#1359]
In the case of the new `bbob-constrained` test suite, the `.info` on a problem of the suite give an uninformative string such as
```
bbob-constrained_f01_i01_d02 single-objective problem (bbob-constrained_f001_i01_d02__bbob-
constrained_f001_i01_d02(0))
```
instead of, e.g., stating the number of constraints in the message. Or is this intended?

**Comment by nikohansen** [comment]
The implementation of the `info` property is generic and IMHO should be. That is, it does not depend on the specific suite. Otherwise, I do not see any intentions, just that the use case never came up.

FTR, it is simple to access the number of constraints via attribute `number_of_constraints`.

**Comment by nikohansen** [comment]
addressed with commit a0eb61cab0d97e2c8257c1401a4cf25c9a97256e in implementation of `__str__` method.

**Comment by nikohansen** [comment]
FTR, the `.info` attribute basically provides the information the underlying C-functions provide. If there is anything is missing or should be added we should open a new issue.
